src/Robo/Commands/Deploy/DeployCommand.php

```diff
@@ -653,4 +666,59 @@ public function installDrupal() {
     ]);
   }
 
+  /**
+   *
+   * Truncates the history from the deployment branch on the remote repo.
+   * This reduces the number of objects in git and drastically reduces
+   * deployment times on much larger projects.
+   * Trigger with --truncate-history option
+   *
+   * @param $options
+   *
+   * @return bool
+   * @throws \Acquia\Blt\Robo\Exceptions\BltException
+   */
+  public function truncateBuildRepoHistory($options) {
+
+    if ($options['truncate-history']) {
```

Comment:
I would test in one place only.
I would probably keep it here.

Reply:
I've removed the second check